task1/jdbc: Class that implements basic queries in database

qnislavnachev opened the pull request:
There is a class that implements most basic queries which contains the
following commands: insert, update, delete, select, alter, drop, where,
like.

qnislavnachev commented:
This pull request is not ready for review yet , because I have to made some changes.

qnislavnachev commented:
@mgenov rewriten test now they are isolated

mgenov commented on src/task1/Queries.java:
```diff
@@ -0,0 +1,68 @@
+        statement = connection.createStatement();
+    }
+
+    public void printTable() throws SQLException {
```
printTable is common name but it uses concrete fields "Name", "Age", "Course" which is wrong. 

The name of the object should be common or concrete, but not both. 

The design should be:
```java
public interface EmployeeRepository {

  void register(Employee employee);

  List<Employee> findEmployees();
}
```

or 

```java
public interface Tables {
  // list all table rows
  List<Object[]> listTableRows(String tableName);

  // returns  updated rows
  int update(String tableName, String[] columns, Object[] values); 

}
```

The test should be updated depending on the used structure.

qnislavnachev replied:
I will fix that

qnislavnachev commented on test/queriestest/TestQueries.java:
```diff
@@ -0,0 +1,144 @@
+    public void deleteInfo() throws Exception {
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(out));
+        queries.insert(1, "Iani", 23, 3);
```
It will be fixed

qnislavnachev commented on test/queriestest/TestQueries.java:
```diff
@@ -0,0 +1,144 @@
+
+    @Test
+    public void addColumn() throws Exception {
+        queries.addColumn("Location", "varchar(20)");
```
Null value

qnislavnachev commented on test/queriestest/TestQueries.java:
```diff
@@ -0,0 +1,144 @@
+
+    @Test
+    public void dropTable() throws Exception {
+        queries.dropTable();
```
that will be fixed

qnislavnachev commented:
last fixes done renamed methods now they are more understandable

mgenov commented on src/task1/StudentsRepository.java:
```diff
@@ -0,0 +1,70 @@
+import java.util.List;
+
+public class StudentsRepository {
+    private Statement statement;
```
For each operation you need a statement and this class is having a few.

mgenov commented on test/queriestest/TestQueries.java:
```diff
@@ -0,0 +1,141 @@
+    @Test
+    public void insertInfoToStudents() throws Exception {
+        Student iani = new Student(1, "Iani", 23, 2);
+        students.register(iani);
```
```java
students.register(iani);
List<Student> allStudents = students.findAllStudents();
assertThat(allStudents.size(), is(1));
assertThat(allStudents.get(0).name, is(equalTo("Iani"));
```

qnislavnachev commented:
I made some refactoring

mgenov commented:
What was Refactored ? You will save me time If I know where to look.

qnislavnachev commented:
renamed last commit because it was wrong named, this commit contain queries method changes, no longer they throws sqlexception and removed result set from most of tests

qnislavnachev commented:
some little changes made now deleting and updating info by person ID
